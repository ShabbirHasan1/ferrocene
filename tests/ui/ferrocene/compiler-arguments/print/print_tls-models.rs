// check-pass
// compile-flags: --print tls-models

fn main() {}

// ferrocene-annotations: um_rustc_print
