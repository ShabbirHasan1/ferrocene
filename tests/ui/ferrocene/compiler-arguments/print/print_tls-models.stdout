Available TLS models:
    global-dynamic
    local-dynamic
    initial-exec
    local-exec
    emulated

